src/device-registry/config/global/db-projections.js

```diff
@@ -485,6 +486,15 @@ const dbProjections = {
       "cohorts.createdAt": 0,
       "cohorts.updatedAt": 0,
       "cohorts.__v": 0,
+      "grids.shape": 0,
```

Comment:
Also since grids have so much details, I had to ensure that I reduce the content returned so that we can optimise the performance of the endpoint. @BenjaminSsempala